templates/dashboard/includes/_filters.html

```diff
@@ -9,7 +9,7 @@
       <div class="col s12">
         <span class="card-title">{% trans 'Filters' context 'Dashboard card title' %}</span>
         {{ filter.get_summary_message }}
-        <span class="collapse-activate hide-on-large-only"><svg data-src="{% static '/images/filter-icon.svg' %}" /></span>
+        <span class="collapse-activate hide-on-large-only"><svg data-src="{% static 'images/filter-icon.svg' %}" /></span>
```

Comment:
I think this happens in few other places as well, if you look for ` "/` or ` '/` in .html files, do you mind fixing it as well?

Reply:
You are right, I forgot to check both. :+1: